app/models/mq/assessment.rb

```diff
@@ -15,7 +15,7 @@
 
 module MQ
   class Assessment < ApplicationRecord
-    LEVELS = %w[1 2 3 4 4STAR].freeze
+    LEVELS = %w[0 1 2 3 4 4STAR].freeze
```

Comment:
what does `4STAR` mean here?

Reply:
4STAR is like 5 level, they named it like this, we can rename it to 5, but then we will have to take care of mapping that back and forth

Reply:
so maybe just add a comment above, cause this looks like some typo ;)